tests/tests_integration/test_api/test_spaces.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import pytest
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes import Space, SpaceList
+
+
+@pytest.fixture(scope="function")
+def cdf_spaces(cognite_client):
+    yield cognite_client.models.spaces.list(limit=-1)
```

Comment:
Tests will be pretty useless if there are no spaces present. Let's make sure there are.
```suggestion
    spaces = cognite_client.models.spaces.list(limit=-1)
    assert len(spaces) > 0
    yield spaces
```